### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shounak-mulay**

### CODE REVIEW

**Comment by shounak-mulay** [comment]
Depends on #23

**Comment by gtxtreme** [comment]
You've included Flutter related README.md by mistake
Please check once

**Comment by shounak-mulay** [comment]
Only the flavor and CI CD part are from flutter template. I have updated all the other parts for Android.

**Comment by gtxtreme** [comment]
@shounak-mulay check the README again

File: README.md [hunk 1]
```diff
@@ -120,39 +120,32 @@ Intent is any action takes place on fragment. It may or may not be user initiate
...
-
-To run a specific falvor you need to specify the flavor and target file.
-```shell
- flutter run --flavor qa -t lib/entrypoints/main_qa.dart
-```
-
-**To avoid specifying all the flags every time, use the [`run.sh`](scripts/README.md#run) script**
+| Note: Flavour specific configurations can be made in the [app.gradle](app/app.gradle) file under the `buildTypes` block
 
 Read the [scripts documentation](scripts/README.md) to learn about all the scrips used in the project.
```
Comment (shounak-mulay) [inline README.md:142]: I think we can remove this line since `Flavors` block is not referencing any scripts.

File: README.md [hunk 2]
```diff
@@ -162,36 +155,41 @@ The Flutter Template contains:
...
+- Save the `store password` in github secrets with key name `KEYSTORE_PASSWORD`.
+- Save the `key alias` in github secrets with key name `KEY_ALIAS`.
+- Save the `key password` in github secrets with key name `KEY_PASSWORD`.
+- [Create a distribution on app center](https://docs.microsoft.com/en-us/appcenter/distribution/) and get the upload key. You can get it from from [Settings](https://appcenter.ms/settings).
+- Save the app center upload key on github secrets with key name `APP_CENTER_USER_API_TOKEN`.
+- Save the group name as `GROUP_NAME` inside github secrets. Example: `Testers`
+- Save the organisation name as `ORG_NAME` inside github secrets. Example: `Wednesday Solutions`
+- Save the app name as `APP_NAME` inside github secrets. Example: `Android Template`
+
+| Caution: Respect the of the value inside secrets or else AppCenter APIs might have problems looking for your app
```
Comment (shounak-mulay) [inline README.md:195]: Something seems to be incorrect here.

Comment (gtxtreme) [reply README.md:195]: Resolved by changing the `I` to `>`, I actually intended to write a block

File: README.md (hunk 1 above)
Comment (gtxtreme) [reply README.md:142]: I don't get it, can you specify what exactly you want changed?

**Review by gtxtreme — APPROVED** [review]

**Pull request merged**